_layouts/default.md

```diff
@@ -67,9 +67,12 @@
 <hr />
 <div>
   <h4 class="text-center text-muted">powered by <img src="{{ site.baseurl }}/app/logo.png" class="mr-2" style="height: 1.2em;" /></h4>
-  <p class="text-center text-muted mb-0" style="font-size: 0.8em">Copyright © Form.io LLC 2019. All rights reserved</p>
+  <p class="text-center text-muted mb-0" style="font-size: 0.8em">Copyright © Form.io LLC 2020. All rights reserved</p>
 </div>
 <p class="text-center text-muted mb-0" style="font-size: 0.8em">Renderer v<span id="renderer-version"></span></p>
+{% if page.noFormio %}
```

Comment:
But what this says is you only are going to include the renderer if they have "noFormio" provided in the page... this means the only page that will include this is the inline embed page.

Reply:
I will fix this myself.  We need to get a build going.